test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -138,6 +138,19 @@ export default class BaseFormObject {
     });
   }
 
+  getAllButtons() {
+    return within(this.el).getAllByRole('button');
+  }
+
+  /**
+ * @param {string} name the text of the button to click
+ */
```

Comment:
true nitty nit: JSDoc whitespace (since it might not syntax color correctly if it's not spaced properly)